scripts/measures/build-measures

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env bash
 
+year=2018
```

Comment:
Not a huge deal but should this be 2017 until 2018 data is added?

Reply:
Yes it should logically... but I just found out I need to add the generated 2018 files in this PR for the tests to run properly, so I'll leave this as 2018-- 2018 data coming tomorrow! :)